Sprint 1/Harmony/Web.config

```diff
@@ -10,7 +10,7 @@
   </configSections>
   <!-- Enable it if you need to test it locally; Disable it if you need to test it on the cloud -->
   <connectionStrings>
-    <!--<add name="HarmonyContext" connectionString="data source=(localDB)\MSSQLlocalDB;initial catalog=Harmony;integrated security=True;MultipleActiveResultSets=True;App=EntityFramework" providerName="System.Data.SqlClient" />-->
+    <add name="HarmonyContext" connectionString="Data Source=(LocalDB)\MSSQLLocalDB;AttachDbFilename='C:\Users\nmgei\seniorproject-PacificWhales\Sprint 1\Harmony\App_Data\db_harmony.mdf';Integrated Security=True;MultipleActiveResultSets=true" providerName="System.Data.SqlClient" />
```

Comment:
Next time just add your own connectionstring to it. Don't delete or replace the commented one.